### PR TITLE
updated CONTRIBUTING.md, made it way better, precise, fun and emojified!

### PR DESCRIPTION
**Pull request by 0Armaan025**
Hey there!
First, if you are reading this , you are amazing⭐

I'm Armaan and taking part in Hacktoberfest and Hacksquad, many folks were contributing to this repo, so I thought why not as well.

When I checked the "CONTRIBUTING.md", it looked amazing yet simple.

I thought why not level it up as well? 

So, I made "CONTRIBUTING.md" way better, precise, fun and emojified! 🔥

Since I'm not the owner/repo maintainer, there might be a few problems in that and I deeply apologise for that.

This pr is free for any edits/modifications.

Additional INFO: Starred⭐✔️

Thanks and regards,
Armaan.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -11,57 +11,57 @@ Please take a moment to review the following guidelines to ensure a smooth and c
...
+5. **Test Your Changes:**
+   - Cast a spell of testing to ensure your changes function correctly without introducing any gremlins.
 
-6. Add your work in the README.md file using our [Style Guide](README.md#style-guide).
+6. **Add Your Work to README:**
+   - Weave your masterpiece into the README.md following our [Style Guide](README.md#style-guide).
 
-6. Commit Your Changes:
-   - Commit your changes with a meaningful commit message.
+6. **Commit Your Changes:**
```
Comment (aayush105) [inline CONTRIBUTING.md:40]: Repeated "no 6" fix this and push it again @0Armaan025 .

Comment (0Armaan025) [reply CONTRIBUTING.md:37]: Fixed

Comment (aayush105) [reply CONTRIBUTING.md:37]: You removed one point from the CONTRIBUTING.md file i.e add you work in README.md, don't do that just change the number(no) of the points.

Comment (0Armaan025) [reply CONTRIBUTING.md]: fixed it @aayush105 , thank you for pointing it out!

**Review by aayush105 — APPROVED** [review]
LGTM!
Thank you @0Armaan025  for contributing.

**Pull request merged**